recipes/gnu-getopt/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "gnu-getopt" %}
+{% set version = "1.1.6" %}
+{% set md5 = "4eaff9e6d8b2504a2e0d29ff0e9fd956" %}
+
+package: 
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz  # [osx]
+  md5: {{ md5 }}  # [osx]
+  url: http://frodo.looijaard.name/system/files/software/getopt/getopt-{{ version }}.tar.gz  # [osx]
+  patches:
+    - Makefile.osxpatch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [not osx]
+
+requirements:
+  build:
+    - gcc  # [osx]
```

Comment:
Please replace with `toolchain`.

Reply:
Sure, but where is this documented? It is not part of the linter.